KeeAgent/ExtensionMethods.cs

```diff
@@ -362,7 +362,7 @@ static byte[] GetPassphrase(this PwEntry entry)
       return Encoding.UTF8.GetBytes(passphrase);
     }
 
-    public static ISshKey GetSshKey(this PwEntry entry)
+    public static ISshKey GetSshKey(this PwEntry entry, KeeAgentExt ext)

```

Comment:
Instead of passing the entire extension, can we just have a boolean parameter for enabling/disabling UI?

Reply:
@dlech , sure